### PR TITLE
wagmi connector for web3auth

### PR DESCRIPTION
**Pull request by himanshuchawla009**

### CODE REVIEW

File: src/lib/connector.ts [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+
+  private loginModal: LoginModal;
+
+  private socialLoginAdapter: OpenloginAdapter;
+
+  constructor(config: { chains?: Chain[]; options: Options }) {
+    super(config);
+    this.web3authOptions = config.options;
+    const chainId = config.options.chainId ? parseInt(config.options.chainId, 16) : 1;
+    const chainConfig = this.chains.filter((chain) => {
```
Comment (chaitanyapotti) [inline src/lib/connector.ts]: just use this.chains.filter(x => x.id === chainId)

**Comment by chaitanyapotti** [comment]
TODO:
- Client id support
- incorrect casing

**Comment by himanshuchawla009** [comment]
> TODO:
> 
> * Client id support
> * incorrect casing

done

**Review by chaitanyapotti — APPROVED** [review]
lgtm

**Pull request merged**